shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -101,6 +101,9 @@ void WebFrameMain::MarkRenderFrameDisposed() {
 void WebFrameMain::UpdateRenderFrameHost(content::RenderFrameHost* rfh) {
   // Should only be called when swapping frames.
   render_frame_disposed_ = false;
+  LOG(INFO) << "old rfh token: " << render_frame_->GetFrameToken();
+  LOG(INFO) << "new rfh token: " << rfh->GetFrameToken();
+  LOG(INFO) << "rfhs equal " << (render_frame_ == rfh ? "true" : "false");
```

Comment:
This looks like it might be accidental leftover from a debugging session?

If this was intentional, no worries -- just asking in case it is accidental